app-office/calligra/calligra-5.9999.ebuild

```diff
@@ -0,0 +1,244 @@
+		webkit
+	)
+	colorio? ( $(cfeat krita) )
+	okular? ( $(cfeat stage) )
+	qtquick? ( $(cfeat stage) )
+"
+
+COMMON_DEPEND="
+	$(add_frameworks_dep kactivities)
+	$(add_frameworks_dep karchive)
+	$(add_frameworks_dep kcodecs)
+	$(add_frameworks_dep kcompletion)
+	$(add_frameworks_dep kconfig)
+	$(add_frameworks_dep kconfigwidgets)
+	$(add_frameworks_dep kcoreaddons)
+	$(add_frameworks_dep kdelibs4support)
+	$(add_frameworks_dep kguiaddons)
+	$(add_frameworks_dep ki18n)
+	$(add_frameworks_dep kiconthemes)
+	$(add_frameworks_dep kio)
+	$(add_frameworks_dep kitemviews)
+	$(add_frameworks_dep kservice)
+	$(add_frameworks_dep ktextwidgets)
+	$(add_frameworks_dep kwallet)
+	$(add_frameworks_dep kwidgetsaddons)
+	$(add_frameworks_dep kwindowsystem)
+	$(add_frameworks_dep kxmlgui)
+	$(add_frameworks_dep sonnet)
+	dev-libs/boost
+	dev-qt/qtgui:5
+	dev-qt/qtopengl:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtquick1:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	media-libs/libpng:0
+	sys-libs/zlib
+	colorio? ( media-libs/opencolorio )
+	crypt? ( >=app-crypt/qca-2.1.0:2[qt5] )
+	dbus? ( dev-qt/qtdbus:5 )
+	diagrams? ( $(add_kdeapps_dep kdiagram) )
+	exif? ( >=media-gfx/exiv2-0.16:= )
+	fontconfig? ( media-libs/fontconfig )
+	gsl? ( sci-libs/gsl )
+	import-filter? (
+		app-text/libetonyek
+		app-text/libodfgen
+		app-text/libwpd:0.10
+		app-text/libwpg:0.3
+		app-text/libwps
+		dev-libs/librevenge
+		media-libs/libvisio
+	)
+	jpeg2k? ( media-libs/openjpeg:0 )
+	lcms? ( >=media-libs/lcms-2.4:2 )
+	openexr? ( media-libs/openexr )
+	opengl? ( virtual/opengl )
+	okular? ( $(add_kdeapps_dep okular) )
+	pdf? ( >=app-text/poppler-0.28:=[qt5] )
+	phonon? ( media-libs/phonon[qt5] )
+	raw? ( $(add_kdeapps_dep libkdcraw) )
+	spacenav? ( dev-libs/libspnav )
+	sql? ( dev-qt/qtsql:5 )
+	truetype? ( media-libs/freetype:2 )
+	webkit? ( dev-qt/qtwebkit:5 )
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	$(add_frameworks_dep khtml)
+	$(add_frameworks_dep kjobwidgets)
+	$(add_frameworks_dep knotifications)
+	$(add_frameworks_dep knotifyconfig)
+	dev-qt/qtnetwork:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtx11extras:5
+	media-libs/glew
+	media-libs/glu
+	media-libs/ilmbase
+	media-libs/libjpeg-turbo
+	media-libs/tiff
+	sci-libs/fftw
+	x11-libs/libX11
+	x11-libs/libXi
+"
+# declarative and kactivities are optional though
+DEPEND="
+	${COMMON_DEPEND}
+	$(add_frameworks_dep kdeclarative)
+	$(add_frameworks_dep kemoticons)
+	$(add_frameworks_dep kitemmodels)
+	$(add_frameworks_dep kross)
+	$(add_frameworks_dep kparts)
+	$(add_frameworks_dep threadweaver)
+	dev-lang/perl
+	dev-libs/libical
+	>=x11-misc/shared-mime-info-0.18
+	eigen? ( dev-cpp/eigen:3 )
+	iconv? ( virtual/libiconv )
+	icu? ( dev-libs/icu:= )
+	kdepim? (
+		$(add_kdeapps_dep kcalcore)
+		$(add_kdeapps_dep kcontacts)
+		$(add_kdeapps_dep kdepimlibs)
+	)
+	vc? ( dev-libs/vc )
+"
+
+#[[ ${PV} == 9999 ]] && LANGVERSION="2.4" || LANGVERSION="$(get_version_component_range 1-2)"
+#PDEPEND=">=app-office/calligra-l10n-${LANGVERSION}"
```

Comment:
What about translations? Are they included into calligra repo? If yes drop the commented lines, otherwise we need a solution for this.